pytorch_lightning/logging/comet_logger.py

```diff
@@ -1,25 +1,94 @@
 try:
     from comet_ml import Experiment as CometExperiment
+    from comet_ml.papi import API
 except ImportError:
     raise ImportError('Missing comet_ml package.')
 
+from torch import is_tensor
+
 from .base import LightningLoggerBase, rank_zero_only
 
 
 class CometLogger(LightningLoggerBase):
-    def __init__(self, *args, **kwargs):
-        super(CometLogger, self).__init__()
-        self.experiment = CometExperiment(*args, **kwargs)
+    def __init__(self, api_key, workspace, rest_api_key=None, project_name=None, experiment_name=None, **kwargs):
+        """
+        Initialize a Comet.ml logger
+
+        :param api_key: API key, found on Comet.ml
```

Comment:
it would be nice to write a data type...
`:param {data-type} workspace: Name of workspace for this user`
just an example:
`:param str workspace: Name of workspace for this user`